index.bs

```diff
@@ -114,224 +112,272 @@ interface ImageCapture {
-  <dd>This reflects the current <a>sharpness</a> setting of the camera and permitted range. Values are numeric. Increasing values indicate increasing sharpness, and the minimum value always implies no sharpness enhancement or processing.</dd>
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>sharpness</code></dfn></dt>
+  <dd>This reflects the permitted <a>sharpness</a> range of the camera. Values are numeric. Increasing values indicate increasing sharpness, and the minimum value always implies no sharpness enhancement or processing.</dd>
 
-  <dt><dfn attribute for="PhotoCapabilities"><code>imageHeight</code></dfn></dt>
-  <dd>This reflects the image height range supported by the UA and the current height setting.</dd>
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>imageHeight</code></dfn></dt>
+  <dd>This reflects the <a>image height</a> range supported by the UA.</dd>
 
-  <dt><dfn attribute for="PhotoCapabilities"><code>imageWidth</code></dfn></dt>
-  <dd>This reflects the image width range supported by the UA and the current width setting.</dd>
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>imageWidth</code></dfn></dt>
+  <dd>This reflects the <a>image width</a> range supported by the UA.</dd>
 
-  <dt><dfn attribute for="PhotoCapabilities"><code>zoom</code></dfn></dt>
-  <dd>This reflects the zoom value range supported by the UA and the current zoom setting.</dd>
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>zoom</code></dfn></dt>
+  <dd>This reflects the <a>zoom</a> value range supported by the UA.</dd>
 
-  <dt><dfn attribute for="PhotoCapabilities"><code>fillLightMode</code></dfn></dt>
-  <dd>This reflects the current fill light (flash) mode setting.  Values are of type {{FillLightMode}}.</dd>
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>fillLightMode</code></dfn></dt>
+  <dd>A sequence of supported <a>fill light mode</a> {{FillLightMode}}s settings.</dd>
 </dl>
 
+## {{MediaTrackConstraintSet}} dictionary ## {#mediatrackconstraintset-section}
+
+{{MediaTrackConstraintSet}} dictionary is used for both reading the current status with {{getConstraints()}} and for applying a set of constraints with {{applyConstraints()}}.
+
 <div class="note">
-  The supported resolutions are presented as segregated {{imageWidth}} and {{imageHeight}} ranges to prevent increasing the fingerprinting surface and to allow the UA to make a best-effort decision with regards to actual hardware configuration.
+{{MediaTrackSettings}} can be retrieved to verify the effect of the application by the user agent of the requested {{MediaTrackConstraints}}. Some constraints such as, e.g. <a>zoom</a>, might not be immediately applicable.
 </div>
 
-## Discussion
-<em>This section is non-normative.</em>
+<pre class="idl">
+  partial dictionary MediaTrackConstraintSet {
+    MeteringMode       whiteBalanceMode;
+    ConstraintDouble   colorTemperature;
```

Comment:
Corrected. The link was working because I defined it at the bottom of the file, and the `text` section was wrong while the `url` was right:
> `urlPrefix: https://www.w3.org/TR/mediacapture-streams/#; type: interface; text: ConstrainDouble ; url: idl-def-ConstrainDouble
`